hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/output/OutputCommitterUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.hive.bigquery.connector.output;
+
+import com.google.cloud.hive.bigquery.connector.JobDetails;
+import com.google.cloud.hive.bigquery.connector.config.HiveBigQueryConfig;
+import com.google.cloud.hive.bigquery.connector.output.direct.DirectOutputCommitter;
+import com.google.cloud.hive.bigquery.connector.output.indirect.IndirectOutputCommitter;
+import com.google.cloud.hive.bigquery.connector.utils.JobUtils;
+import com.google.cloud.hive.bigquery.connector.utils.JobUtils.CleanUp;
+import com.google.cloud.hive.bigquery.connector.utils.hive.HiveUtils;
+import com.google.common.collect.Sets;
+import java.io.IOException;
+import java.util.Set;
+import org.apache.hadoop.conf.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OutputCommitterUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(OutputCommitterUtils.class);
+
+  public static void commitJob(Configuration conf, JobDetails jobDetails) throws IOException {
+    HiveBigQueryConfig opts = HiveBigQueryConfig.from(conf, jobDetails.getTableProperties());
+    String writeMethod = opts.getWriteMethod();
```

Comment:
since all here needs is the write method, maybe instead of converting to HiveBigQueryConfig, just get the write method directly?

Reply:
I made the change. Is that what you were thinking?

Reply:
yes.